tools/pv_tree_generator/util.py

```diff
@@ -15,20 +15,110 @@
 # limitations under the License.
 
 """utility functions used to build the property-value tree structure """
+from google.protobuf import text_format
+import datacommons as dc
 import stat_config_pb2
 import collections
-from google.protobuf import text_format
+from configmodule import ProductionConfig
```

Comment:
please use "DevelopmentConfig here

Reply:
Using the "DevelopmentConfig" raises the following error:
ValueError: Response error 403:
b'{\n "code": 7,\n "message": "API api.datacommons.org is not enabled for the consumer project.",\n "details": [\n  {\n   "@type": "type.googleapis.com/google.rpc.DebugInfo",\n   "stackEntries": [],\n   "detail": "service_control"\n  }\n ]\n}\n'

Reply:
done.